packages/nimble-components/src/utilities/unit-format/decimal-unit-format.ts

```diff
@@ -53,6 +54,37 @@ export class DecimalUnitFormat extends UnitFormat<DecimalUnitFormatOptions> {
         this.maximumFractionDigits = maximumFractionDigits;
     }
 
+    public static normalizeAndDefaultFractionDigitOptions(
+        fractionDigits?: number,
+        minimumFractionDigits?: number,
+        maximumFractionDigits?: number
+    ): {
+            minimumFractionDigits?: number,
```

Comment:
Based on the implementation, it doesn't look like the returned value for `minimumFractionDigits` would ever be `undefined`.